src/main/java/Duke.java

```diff
@@ -232,10 +237,16 @@ private static void executeCommand(CommandType commandType, String userInput, Ar
     private static void addEvent(String userInput) {
         boolean placementCorrect = true;
 
```

Comment:
"placementCorrect" boolean is not phrased as a boolean. Consider "isPlacementCorrect" instead